tests/model_selection/test_cross_validate.py

```diff
@@ -371,5 +371,172 @@ def test_happy_path_with_intersection(
             ],
             "metrics": expected_metrics,
         }
+        assert actual == expected
 
+    @pytest.mark.parametrize(
+        "validate_ref_models,expected_metrics,compute_timings",
```

Comment:
Let's improve it more and make it more clear.
You have `validate_ref_models` and `expected_metrics` depending on each other.
And you have `compute_timings` that has nothing to do with them. But you duplicate same values of previous parameters.
To stop duplicating code just move  `compute_timings` to different `@pytest.mark.parametrize` decorator.

Reply:
done